sources/packages/backend/apps/queue-consumers/src/processors/models/processors.models.ts

```diff
@@ -81,6 +82,24 @@ export class QueueProcessSummary {
       errors: this.errors.length ? this.errors : undefined,
     };
   }
+
+  /**
+   * Writes all the log entries.
+   * @param processSummary process summary logs.
+   * @param logger optionally provides a job logger
+   * in case one is not present.
+   */
+  async logProcessSummaryToJobLogger(
+    processSummary: ProcessSummary,
+    logger?: JobLogger,
+  ): Promise<void> {
+    const jobLogger = this.loggers?.jobLogger ?? logger;
```

Comment:
shouldn't this be reverse, `const jobLogger = logger ??  this.loggers?.jobLogger`